src/base/core/owl_graph.ml

```diff
@@ -3,101 +3,75 @@
  * Copyright (c) 2016-2019 Liang Wang <liang.wang@cl.cam.ac.uk>
  *)
 
-
-type 'a node = {
-  mutable id   : int;            (* unique identifier *)
-  mutable name : string;         (* name of the node *)
-  mutable prev : 'a node array;  (* parents of the node *)
-  mutable next : 'a node array;  (* children of the node *)
-  mutable attr : 'a;             (* indicate the validity *)
-}
-
-type order = BFS | DFS
-
-type traversal = PreOrder | PostOrder
-
-type dir = Ancestor | Descendant
-
+type 'a node =
+  { mutable id : int
+  ; (* unique identifier *)
+    mutable name : string
+  ; (* name of the node *)
+    mutable prev : 'a node array
+  ; (* parents of the node *)
+    mutable next : 'a node array
+  ; (* children of the node *)
+    mutable attr : 'a (* indicate the validity *)
+  }
+
+type order =
+  | BFS
+  | DFS
+
+type traversal =
+  | PreOrder
+  | PostOrder
+
+type dir =
+  | Ancestor
+  | Descendant
 
 let _global_id = ref 0
-
-
 let id x = x.id
```

Comment:
I think it would be better if these space lines between functions could be kept, but perhaps it's just my personal preference.

Reply:
I don't really see a way of changing that changing .ocamlformat at the moment, but I do like how compact it is at the moment